packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.base.tsx

```diff
@@ -104,8 +105,9 @@ export class CheckboxBase extends React.Component<ICheckboxProps, ICheckboxState
     return (
       <KeytipData keytipProps={keytipProps} disabled={disabled}>
         {(keytipAttributes: any): JSX.Element => (
-          <div className={this._classNames.root} title={title}>
+          <div role="presentation" className={this._classNames.root} title={title}>
```

Comment:
>title={title} [](start = 69, length = 13)

Why does this have a title? Having both title and role=presentation doesn't make sense... Really, what is this element?

Reply:
the element is just a wrapper. I added 'presentation' because I wanted to ensure that the input was representing the proper role.

Reply:
but this is going to make it show read out as "<title> group" This doesn't need role=presentation because screen readers are smart enough now that they don't add positioning elements to the accessiblity tree unless they have an accessible name (which this element is currently getting because it has a title... hence that also needs to be removed and then the element won't be represented in the accessiblity tree as expected)